doc/rtd/topics/faq.rst

```diff
@@ -317,33 +317,51 @@ Where can I learn more?
 Below are some videos, blog posts, and white papers about cloud-init from a
 variety of sources.
 
+Videos:
+
 - `cloud-init - The Good Parts`_
-- `cloud-init Summit 2019`_
-- `Utilising cloud-init on Microsoft Azure (Whitepaper)`_
-- `Cloud Instance Initialization with cloud-init (Whitepaper)`_
-- `cloud-init Summit 2018`_
-- `cloud-init - The cross-cloud Magic Sauce (PDF)`_
-- `cloud-init Summit 2017`_
+- `Perfect Proxmox Template with Cloud Image and Cloud Init [proxmox, cloud-init, template]`_
 - `cloud-init - Building clouds one Linux box at a time (Video)`_
-- `cloud-init - Building clouds one Linux box at a time (PDF)`_
 - `Metadata and cloud-init`_
-- `The beauty of cloud-init`_
 - `Introduction to cloud-init`_
 
+Blog Posts:
+
+- `cloud-init - The cross-cloud Magic Sauce (PDF)`_
+- `cloud-init - Building clouds one Linux box at a time (PDF)`_
+- `The beauty of cloud-init`_
+- `Cloud-init Getting Started [fedora, libvirt, cloud-init]`_
+- `Build Azure Devops Agents With Linux cloud-init for Dotnet Development [terraform, azure, devops, docker, dotnet, cloud-init]`_
+- `Cloud-init Getting Started [fedora, libvirt, cloud-init]`_
+- `Setup Neovim cloud-init Completion [neovim, yaml, Language Server Protocol, jsonschema, cloud-init]`_
+
+Events:
+
+- `cloud-init Summit 2019`_
+- `cloud-init Summit 2018`_
+- `cloud-init Summit 2017`_
+
+
+Whitepapers:
+
+- `Utilising cloud-init on Microsoft Azure (Whitepaper)`_
+- `Cloud Instance Initialization with cloud-init (Whitepaper)`_
+
 .. _cloud-init - The Good Parts: https://www.youtube.com/watch?v=2_m6EUo6VOI
-.. _cloud-init Summit 2019: https://powersj.io/post/cloud-init-summit19/
 .. _Utilising cloud-init on Microsoft Azure (Whitepaper): https://ubuntu.com/engage/azure-cloud-init-whitepaper
 .. _Cloud Instance Initialization with cloud-init (Whitepaper): https://ubuntu.com/blog/cloud-instance-initialisation-with-cloud-init
-.. _cloud-init Summit 2018: https://powersj.io/post/cloud-init-summit18/
+
 .. _cloud-init - The cross-cloud Magic Sauce (PDF): https://events.linuxfoundation.org/wp-content/uploads/2017/12/cloud-init-The-cross-cloud-Magic-Sauce-Scott-Moser-Chad-Smith-Canonical.pdf
-.. _cloud-init Summit 2017: https://powersj.io/post/cloud-init-summit17/
 .. _cloud-init - Building clouds one Linux box at a time (Video): https://www.youtube.com/watch?v=1joQfUZQcPg
 .. _cloud-init - Building clouds one Linux box at a time (PDF): https://annex.debconf.org/debconf-share/debconf17/slides/164-cloud-init_Building_clouds_one_Linux_box_at_a_time.pdf
```

Comment:
`annex.debconf.org` is [down](https://www.mail-archive.com/debconf-team@lists.debconf.org/msg14555.html). I have tried to find where they moved the slides without success. What do you think about offering a Wayback-Machine link?

```suggestion
.. _cloud-init - Building clouds one Linux box at a time (PDF): https://web.archive.org/web/20181111020605/https://annex.debconf.org/debconf-share/debconf17/slides/164-cloud-init_Building_clouds_one_Linux_box_at_a_time.pdf
```

Reply:
@aciba90 Good catch, thanks! It looks like sphinx has a built-in link checker. I don't think we want in CI, but I'll add a tox target in a followup PR for periodic scrubbing.